i18n/cmd/commands/transifex/pull_transifex.go

```diff
@@ -223,7 +226,7 @@ func getDownloadURL(languageCode, downloadID string) string {
 		case "pending":
 			fallthrough
 		case "processing":
-			fmt.Printf("Current status for language %s: %s\n", languageCode, status)
+			fmt.Printf(tr("Current status for language %s: %s\n"), languageCode, status)
```

Comment:
I don't think it makes sense to internationalize this internal utility package. Doing so would increase the workload for the community translators without any significant benefit for the project users in return.

Credentials in the Transifex project are required to run this, which only a few select administrators would have. This is intended to be ran automatically by the continuous distribution system, so the logs are only ever viewed in the case of the need to troubleshoot a failure. In that rare eventuality, if the person doing the troubleshooting didn't have English language fluency, they could use an automatic translation service (e.g., Google Translate) on the messages.

Reply:
Agree, these messages will never reach any `arduino-cli` user.